cluster-operator/src/test/java/io/strimzi/operator/cluster/KafkaVersionTestUtils.java

```diff
@@ -14,25 +14,28 @@
 public class KafkaVersionTestUtils {
 
     public static final String KAFKA_VERSION_YAML_TEMPLATE =
-                    "- version: %s\n  format: %s\n  protocol: %s\n  checksum: %s\n  third-party-libs: %s\n  default: %b\n";
+                    "- version: %s\n  format: %s\n  protocol: %s\n  zookeeper: %s\n  checksum: %s\n  third-party-libs: %s\n  default: %b\n";
     public static final String KAFKA_IMAGE_STR = "strimzi/kafka:latest-kafka-";
     public static final String KAFKA_CONNECT_IMAGE_STR = "strimzi/kafka-connect:latest-kafka-";
     public static final String KAFKA_CONNECT_S2I_IMAGE_STR = "strimzi/kafka-connect-s2i:latest-kafka-";
     public static final String KAFKA_MIRROR_MAKER_IMAGE_STR = "strimzi/kafka-mirror-maker:latest-kafka-";
 
-    public static final String LATEST_KAFKA_VERSION = "2.3.1";
+    public static final String LATEST_KAFKA_VERSION = "2.3.0";
```

Comment:
Why are you changing this back to 2.3.0? Just curious, maybe you know more about what it does :-o

Reply:
TBH I am not sure I had a very strong reason for changing this back, but the actual values in this class are not really relevant. The idea of this util class is to remove the need for any hardcoded version numbers in the tests.